configs/docker-compose.yaml

```diff
@@ -129,4 +130,4 @@ networks:
      config:
        - subnet: 172.100.0.0/16
```

Comment:
Only thing I can raise is this falls outside of RFC 1918 and is technically used by Spectrum in the US...

Reply:
https://ipinfo.io/ips/172.100.0.0/16